plugins/modules/system/kernel_blacklist.py

```diff
@@ -114,7 +117,7 @@ def main():
         argument_spec=dict(
             name=dict(type='str', required=True),
             state=dict(type='str', default='present', choices=['absent', 'present']),
-            blacklist_file=dict(type='str')
+            blacklist_file=dict(type='path')
```

Comment:
I think this needs a changelog fragment, since for `blacklist_file` now `~` and env vars are replaced (https://docs.python.org/3/library/os.path.html#os.path.expanduser, https://docs.python.org/3/library/os.path.html#os.path.expandvars).

Reply:
I prefer not to go into this level of details now, so I will just undo this.